types/react-native-push-notification/index.d.ts

```diff
@@ -84,6 +84,7 @@ export class PushNotificationObject {
     id?: string | number | undefined;
     title?: string | undefined;
     message: string;
+    picture?: string;
```

Comment:
Judging by the other properties in this object, I'm guessing this needs to be:
```suggestion
    picture?: string | undefined;
```

Reply:
Thanks @jakebailey suggestion committed.